website/sidebars.js

```diff
@@ -429,9 +429,11 @@ const sidebars = {
           label: 'Quickstart (Rust)',
         },
         {
-          type: 'doc',
-          label: 'Run a local dev node',
-          id: 'stylus/how-tos/local-stylus-dev-node',
+          type: 'html',
+          value:
+            '<a class="menu__link menu__list-item" href="/run-arbitrum-node/run-local-dev-node">Run a full Stylus node <span class="other-section-icon">↑</span></a>',
```

Comment:
This says 'Run a full Stylus node' but it links to running a dev node. I think 'full node' to me means running an Arbitrum node that syncs with a live chain.

Reply:
100% agreed! what do you think about replacing` Run a full Stylus node` with `Run a Stylus dev node`?